### PR TITLE
Camera reboot function

### PR DESCRIPTION
**Pull request by cnighswonger**
Dahua cameras support a "Reboot" function. Other's may as well. This series of commits adds a reboot function option to the camera control configuration as well as a reboot button to the control UI.

### CODE REVIEW

File: web/skins/classic/includes/control_functions.php [hunk 1]
```diff
@@ -18,344 +18,389 @@
 // Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 // 
 
-function getControlCommands( $monitor ) {
-  $cmds = array();
+function getControlCommands( $monitor )
+{
```
Comment (mnoorenberghe) [inline web/skins/classic/includes/control_functions.php]: There seems to be a lot of unrelated style changes in this file. Can you revert them?

Comment (cnighswonger) [reply web/skins/classic/includes/control_functions.php]: My apologies. Something is really screwed up with those commits. I'll revert them and recommit the fixes.

Comment (cnighswonger) [reply web/skins/classic/includes/control_functions.php]: Okay. This problem was due to my convoluted workflow. My working branch was based off of an older version of master and so effectively reverted a number of changes made in master since the older version. I've opened pull request #2554 for corrected patches.

Thanks for drawing my attention to a real mess!
